### DISCUSSION TITLE
Writing thumbnail to JPEG

### DISCUSSION THREAD

**Discussion by jefflorenzini** [dlemstra/Magick.NET#1344] [Help] [answered]
I'm rotating a JPEG and I want to rotate the thumbnail in the Exif data to match. I can get the existing thumbnail from the Exif data (which admittedly I don't use at all), I can create the rotated thumbnail image, but I have no idea how to save the new thumbnail image to the JPEG file.

Any ideas? I appreciate any help.

**Comment by dlemstra** [comment, marked as answer]
There currently is no easy api to change the thumbnail of the exif profile.

**Reply by jefflorenzini** [reply]
Bummer. I figured it's a little difficult since it's embedded so tightly in the Exif data, you'd have to recalculate the whole Exif header. The mitigating factor is when I do a lossless JPEG rotate, it also rotates the thumbnail. And most images with an embedded thumbnail will probably also have a height/width divisible by 16 which means I can support lossless rotation on them.

Thanks for responding.
